source/adios2/toolkit/transport/file/FilePointer.cpp

```diff
@@ -48,16 +48,16 @@ void FilePointer::Open(const std::string &name, const OpenMode openMode)
 
     if (m_OpenMode == OpenMode::Write)
     {
-        m_File = std::fopen(name.c_str(), "w");
+        m_File = fopen(name.c_str(), "wb");
```

Comment:
This is the root cause for a corrupted file on Windows. I am switching to fstream binary, fopen triggers a non safe warning in favor of fopen_s